src/init.js

```diff
@@ -41,6 +42,8 @@ function init(options) {
     var fileContent = file.contents.toString();
     var sourceMap;
 
+    file.newline = detectNewline.graceful(fileContent || '');
```

Comment:
Well, it is little complicated to me work with exports in v2.
In v1 things were simpler…